src/js/components/Text/propTypes.js

```diff
@@ -81,7 +81,7 @@ if (process.env.NODE_ENV !== 'production') {
     ]),
     truncate: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['tip'])]),
     weight: PropTypes.oneOfType([
-      PropTypes.oneOf(['normal', 'bold']),
+      PropTypes.oneOf(['normal', 'bold', 'bolder']),
```

Comment:
Recommend we support the [common keyword font-weight values](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#syntax). We have 'normal', 'bold', and numerics, so just need to add 'lighter' and 'bolder'.

```suggestion
      PropTypes.oneOf(['normal', 'bold', 'bolder', 'lighter']),
```